app/assets/javascripts/client/inventories/Inventory.js

```diff
@@ -24,6 +24,27 @@
       'districtProductEntries': {
         method: 'GET',
         url: UrlService.url('inventories/:inventory_id/district_product_entries')
+      },
+      'markComplete': {
+        method: 'POST',
+        url: UrlService.url('inventories/:inventory_id/mark_complete'),
+        params: {
```

Comment:
FYI, I believe you already have `inventory_id` in the URL, so in the rails controller `inventory_id` will still be in `params[:inventory_id]`. Also, the restful way of doing this would be PATCHing the inventory with the right attribute to modify it's state, so `PATCH /inventories/:inventory_id, { response.submitted_at: Time.now}` would have been enough.

We can add this to the cleanup list.